qa/suites/fs/functional/tasks/journal-repair.yaml

```diff
@@ -6,6 +6,7 @@ overrides:
       - Metadata damage detected
       - slow requests are blocked
       - Behind on trimming
+      - error reading sessionmap
```

Comment:
Puzzling why tests didn't fail earlier :-O

Reply:
This is coming from the standby-replay daemon. We only recently started regularly testing standby-replay with fs:workload.